backend/dal/Migrations/20210702193135_v01.10.07.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Pims.Dal.Helpers.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v011007 : SeedMigration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<bool>(
+                name: "SendNotifications",
```

Comment:
We don't want to modify the table, as this would stop all future notifications for this project.  All we really need is an option in the API endpoint and the service layer so that on specific requests it will avoid sending notifications.